dist/select.js

```diff
@@ -862,7 +862,7 @@ uis.controller('uiSelectCtrl',
         }
         break;
       case KEY.TAB:
-        if (!ctrl.multiple || ctrl.open) ctrl.select(ctrl.items[ctrl.activeIndex], true);
+        if (ctrl.open) ctrl.select(ctrl.items[ctrl.activeIndex], true);
```

Comment:
we shouldn't make the change here, this is `dist/` folder, which will be updated when you rebuild the project.

The correct place is line 585 of `uiSelectController.js`. Please revert the change here in dist folder.

Also, you need to build the library as well in this PR:
1. open cli and use node 10
2. run `npm install`
3. run `gulp` to build the change (then there will be changes built to dist/ folder automatically)

Reply:
ah I see, thanks!